.gitignore

```diff
@@ -38,3 +38,6 @@ node_modules
 
 # Vagrant
 /tools/vagrant/.vagrant
+
+# Webstorm
+**/.idea/
```

Comment:
This should probably just be `.idea`. Make sure to remove already commited files from the index, it won't update itself if you ignore files post-commit.